tests/src/string/src/main.sw

```diff
@@ -130,116 +133,107 @@ impl StringTest for Contract {
         string.clear();
         assert(string.is_empty());
 
-        // Uncomment when https://github.com/FuelLabs/sway/issues/4408 is resolved
-        // string.push(NUMBER0);
-        // assert(!string.is_empty());
+        string.push(NUMBER0);
+        assert(!string.is_empty());
 
-        // string.clear();
-        // assert(string.is_empty());
+        string.clear();
+        assert(string.is_empty());
 
-        // string.push(NUMBER0);
-        // string.push(NUMBER1);
-        // string.push(NUMBER2);
-        // string.push(NUMBER3);
-        // string.push(NUMBER4);
-        // string.push(NUMBER5);
-        // string.push(NUMBER6);
-        // string.push(NUMBER7);
-        // string.push(NUMBER8);
-        // assert(!string.is_empty());
+        string.push(NUMBER0);
+        string.push(NUMBER1);
+        string.push(NUMBER2);
+        string.push(NUMBER3);
+        string.push(NUMBER4);
+        string.push(NUMBER5);
+        string.push(NUMBER6);
+        string.push(NUMBER7);
+        string.push(NUMBER8);
+        assert(!string.is_empty());
 
-        // string.clear();
-        // assert(string.is_empty());
+        string.clear();
+        assert(string.is_empty());
 
-        // string.clear();
-        // assert(string.is_empty());
+        string.clear();
+        assert(string.is_empty());
 
-        // string.push(NUMBER0);
-        // assert(!string.is_empty());
+        string.push(NUMBER0);
+        assert(!string.is_empty());
 
-        // string.clear();
-        // assert(string.is_empty());
+        string.clear();
+        assert(string.is_empty());
```

Comment:
This is excessive, I would remove this since we already test the addition and clearing above